app/Http/Controllers/Cv/CvController.php

```diff
@@ -51,47 +58,70 @@ public function create()
      */
     public function store(Request $request)
     {
-        $userID = Auth::id();
-        $user = User::findOrFail($userID);
-        $data = array_merge($request->all(), ["user_id" => $userID]);
+
+        if($request->hasFile('file')) {
+            $path = Storage::putFileAs(
+                'avatars', $request->file('file'), $request->image
+            );
+        }
+
+        $userId = Auth::id();
+        $data = array_merge($request->all(), ["user_id" => $userId]);
         $cv = Cv::create($data);
         $cv->save();
 
-        $nameSkillPro = $request->name_skill_pro;
-        $percentSkillPro = $request->percent_skill_pro;
-        $nameSkillPer = $request->name_skill_per;
-        $percentSkillPer = $request->percent_skill_per;
+        if($request->hasFile('file')) {
+            $nameImageAva = $cv->id.'avatar.jpg';
```

Comment:
@hapo-datnd tên image ko nên lưu theo `id` có hàm `uniqid() function generates a unique ID based on the microtime (current time in microseconds)`

Reply:
Vâng ạ. Để em xem lại ạ!